server/chat-plugins/info.js

```diff
@@ -306,6 +306,33 @@ const commands = {
 		this.sendReplyBox(buf);
 	},
 
+	findsharedbattles(target, room) {
+		if (!this.can('lock')) return false;
+
+		const [targetUsername1, targetUsername2] = target.split(',');
+		const user1 = Users.get(targetUsername1);
+		const user2 = Users.get(targetUsername2);
+		if (!user1) return this.errorReply(`User ${targetUsername1} not found.`);
+		if (!user2) return this.errorReply(`User ${targetUsername2} not found.`);
```

Comment:
This condition shouldn't be required to account for the case when the user has expired (either in the exhaustive version suggested in the comment below, or in the only version if said comment is dismissed).

Reply:
I can't figure out whether tickets outlive the submitter. In that case, ``user1`` shouldn't be checked either.